src/DocumentationBrowserViewExtension/DocumentationBrowserView.xaml.cs

```diff
@@ -142,6 +142,8 @@ async void InitializeAsync()
             }));
 
             this.documentationBrowser.CoreWebView2.WebMessageReceived += CoreWebView2OnWebMessageReceived;
+            this.documentationBrowser.CoreWebView2.Settings.IsZoomControlEnabled = false;
+            this.documentationBrowser.CoreWebView2.Settings.AreDevToolsEnabled = false;
```

Comment:
Any reason we want to disable `AreDevToolsEnabled` only for docs browser?

Reply:
I guess this is to not allow the user to interact with the HTML code. We can either do this or disable the context menu.

Reply:
Context menu is already disabled, but DevTools can still be opened using F12?

Reply:
@QilongTang Yes, it can. But with DevTools disabled, it's not possible to open using F12.

Reply:
@QilongTang @mjkkirschner So, should we consider letting the user open the DevTools?